Formula/npiet.rb

```diff
@@ -0,0 +1,48 @@
+class Npiet < Formula
+  desc "Interpreter for the esoteric, picture-based Piet programming language"
+  homepage "https://www.bertnase.de/npiet"
+  url "https://www.bertnase.de/npiet/npiet-1.3e.tar.gz"
+  sha256 "e819c766d12e1dfaf71561429486ed85b6ae4438da6e7ee06871ab5ce58231c5"
+
+  depends_on "gd"
+  depends_on "libpng"
+  depends_on "homebrew/versions/giflib5"
```

Comment:
5 is the latest. A lot of Homebrew formulae have not been updated to work with 5, however, so Homebrew keeps core as version 4.